File: send.php
<?php
$token = "6018110104:AAHswl_eKT9WMRXzePQbn54Vq-x5uOMUTO4";
$chat_id = "-1001934898535";

function clear_data($val){
  $val = trim($val);
  $val = stripslashes($val);
  $val = htmlspecialchars($val);
  return $val;
}
if(!empty($_POST)){
  $name = clear_data($_POST["name"]);   
  $phone = clear_data($_POST["phone"]);
  $message = clear_data($_POST["message"]);
  $txt = "";
  $arr = [
    'Имя:' => $name,
    'Телефон:' => $phone,
    'Сообщения:' => $message,
  ];
  foreach($arr as $key => $value) {
    $txt .= "<b>".$key."</b> ".$value."%0A";
  }
  if(!empty($token) && !empty($chat_id)){
    $url = "https://api.telegram.org/bot{$token}/sendMessage?chat_id={$chat_id}&parse_mode=html&text={$txt}";
    $sendToTelegram = file_get_contents($url);
    if ($sendToTelegram) {
      echo json_encode('Успешно');
    } else {
     return false;
  
    }
  }
 
}

File: send.py
from telegram.ext import Updater

def get_chat_id(update, context):
    chat_id = update.message.chat_id
    print(chat_id)  # или можно отправить chat_id боту при помощи context.bot.send_message()

updater = Updater(token='6619057216:AAGfdxdAXKNJIu8JrDlvJlrbZVJK1AHTNTI', use_context=True)
updater.dispatcher.add_handler(MessageHandler(Filters.text & (~Filters.command), get_chat_id))

updater.start_polling()